MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs

```diff
@@ -535,8 +585,7 @@ private static bool EnumeratePrefix(SettingsDataFeed __instance, IReadOnlyList<s
                     break;
             }
 
-            var mapper = __instance.Slot.GetComponent((DataFeedItemMapper m) => m.Mappings.Count > 1);
-            mapper?.RunSynchronously(() => EnsureColorXTemplate(mapper));
+            _cachedDataFeedItemMapper = __instance.Slot.GetComponent((DataFeedItemMapper m) => m.Mappings.Count > 1);
```

Comment:
Should this be `>= 1`? Could also use `Mappings.Any()`.

I think I'd use an explicit generic parameter and an implicit lambda, but that's ultimately not super important.

Reply:
There are two DataFeedItemMappers in the Settings facet, one of them has 1 mapping and needs to be ignored, so it needs to be greater than 1.